app/views/landing_pages/contact.html.erb

```diff
@@ -1,20 +1,35 @@
 <div class="container" id="contact">
-<h2>Smart Bet Organizing Committee</h2>	
+<h2>Executive Committee</h2>	
 <p>
-Joe Jablonski - Endstor<br>
-Chris Gladwin - Cleversafe<br>
-Fred Hoch - Illinois Technology Association<br>
-Terry Howerton - TechNexus<br>
-Bart Loethen - Synergy Law Group<br>
-Jim Dugan - OCA Ventures<br>
-<a href="mailto:brendan@technexus.com">Brendan Karp - TechNexus</a>
+Chairman - Joe Jablonski<br/>
+Secretary &amp; Legal - Bart Loethen<br/>
+Treasurer - Jim Dugan<br/>
+Chairman Emeritus – Chris Gladwin<br/>
 </p>
 
-<h2>Smart Bet Web Development</h2>
+<h2>Board Members</h2>
 <p>
-Matt Conway - ClassBug<br>
-Jeano Verschave - Vaystays
+Jeremie Bacon<br/>
+Adam Caplan<br/>
+Brian Crummy<br/>
+Waverly Deutsch<br/>
+Mike Evans<br/>
+Chris Gladwin<br/>
+Fred Hoch<br/>
+Terry Howerton<br/>
+Nik Rokop<br/>
+Greg Rudin<br/>
+Julia Stamberger<br/>
+Chuck Templeton<br/>
 </p>
+
+<h2>Organizing Committee</h2>
+<p>
+Brendan Karp<br/>
+Jeano VerSchave<br/>
+Matt Conway<br/>
```

Comment:
I tried to add these names but it causes a hiccup in the design in that I can't scroll.   @Conhoff, is this something you wouldn't mind taking a look at?

![smartbet-cant-scroll](https://cloud.githubusercontent.com/assets/475206/6111731/b686b1c0-b050-11e4-827d-1e7a701c0faf.jpg)